modules/@apostrophecms/modal/ui/apos/components/AposModal.vue

```diff
@@ -217,6 +235,7 @@ export default {
       // pop doesn't quite suffice because of race conditions when
       // closing one and opening another
       apos.modal.stack = apos.modal.stack.filter(modal => modal !== this);
+      this.focusLastModalFocusedElement();
```

Comment:
when we leave a modal, we want to focus back the stored focused element from the previous modal